packages/components/src/color-palette/README.md

```diff
@@ -1,21 +1,31 @@
 # ColorPalette
 
+ColorPalette allows the user to pick a color from a list of pre-defined color entries.
+
 ## Props
 
 The component accepts the following props.
 
-{ colors, disableCustomColors = false, value, onChange, className, clearable = true }
+{ colors = [],  disableCustomColors = false, enableAlpha, value, onchange, className, clearable = true, __experimentalHasMultipleOrigins = false, __experimentalIsRenderedInSidebar = false }
```

Comment:
```diff
diff --git a/packages/components/src/color-palette/README.md b/packages/components/src/color-palette/README.md
index 49612e3f91..b194f3cef0 100644
--- a/packages/components/src/color-palette/README.md
+++ b/packages/components/src/color-palette/README.md
@@ -36,8 +36,6 @@ for the `ColorPalette`'s color swatches, by rendering your `ColorPalette` with a
 
 The component accepts the following props.
 
-{ colors = [],  disableCustomColors = false, enableAlpha, value, onchange, className, clearable = true, __experimentalHasMultipleOrigins = false, __experimentalIsRenderedInSidebar = false }
-
 ### colors
 Array with the colors to be shown. When displaying multiple color palettes to choose from, the format of the array changes from an array of colors objects, to an array of color palettes.
```

Reply:
Committed in https://github.com/WordPress/gutenberg/pull/44632/commits/29984c0cd55611a12d80b6e2d2cedb5debb2b8bc